### PR TITLE
feat(GraphqlJavaClient): Java Client code using Apollo Library to que…

### PR DESCRIPTION
**Pull request by RajeshSivalingam**
…ry against Graphql API

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.

### CODE REVIEW

File: contrib/JavaGraphqlClient/src/main/graphql/schema.json [hunk 1]
```diff
@@ -0,0 +1 @@
...:null},{"name":"updateChart","description":"Update the metadata about a particular Chart","args":[{"name":"urn","description":null,"type":{"kind":"NON_NULL","name":null,"ofType":{"kind":"SCALAR","name":"String","ofType":null}},"defaultValue":null},{"name":"input","description":null,"type":{"kind":"NON_NULL","name":null,"ofType":{"kind":"INPUT_OBJECT","name":"ChartUpdateInput","ofType":null}},"defaultValue":null}],"type":{"kind":"OBJECT","name":"Chart","ofType":null},"isDeprecated":false,"deprecationReason":null},{"name":"updateDashboard","description":"Update the metadata about a particular Dashboard","args":[{"name":"urn","description":null,"type":{"kind":"NON_NULL","name":null,"ofType":{"kind":"SCALAR","name":"String","ofType":null}},"defaultValue":null},{"name":"input","description":null,"type":{"kind":"NON_NULL","name":null,"ofType":{"kind":"INPUT_OBJECT","name":"DashboardUpdateInput","ofType":null}},"defaultValue":null}],"type":{"kind":"OBJECT","name":"Dashboard","ofType":null},"isDeprecated":false,"deprecationReason":null},{"name":"updateNotebook","description":"Update the metadata about a particular Notebook","args":[{"name":"urn","description":null,"type":{"kind":"NON_NULL"
```
Comment (gabe-lyons) [inline contrib/JavaGraphqlClient/src/main/graphql/schema.json:1]: does it make sense to keep this under version control? won't it immediately become out of date if we update the graphql spec?

Comment (gabe-lyons) [inline contrib/JavaGraphqlClient/src/main/graphql/schema.json:1]: does it make sense to keep this under version control? won't it immediately become out of date if we update the graphql spec?

File: contrib/JavaGraphqlClient/src/test/java/com/example/graphql/client/JavaGraphqlClient/JavaGraphqlClientApplicationTests.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class JavaGraphqlClientApplicationTests {
+
+	@Test
+	void contextLoads() {
+	}
+
+}
```
Comment (gabe-lyons) [inline contrib/JavaGraphqlClient/src/test/java/com/example/graphql/client/JavaGraphqlClient/JavaGraphqlClientApplicationTests.java:13]: could you add an example test here to show how things work?

Comment (gabe-lyons) [inline contrib/JavaGraphqlClient/src/test/java/com/example/graphql/client/JavaGraphqlClient/JavaGraphqlClientApplicationTests.java:13]: could you add an example test here to show how things work?

File: contrib/JavaGraphqlClient/src/main/graphql/schema.json (hunk 1 above)
Comment (RajeshSivalingam) [reply contrib/JavaGraphqlClient/src/main/graphql/schema.json:1]: I thought of keeping it for reference.I remove it

Comment (RajeshSivalingam) [reply contrib/JavaGraphqlClient/src/main/graphql/schema.json:1]: I thought of keeping it for reference.I remove it

File: contrib/JavaGraphqlClient/src/test/java/com/example/graphql/client/JavaGraphqlClient/JavaGraphqlClientApplicationTests.java (hunk 2 above)
Comment (RajeshSivalingam) [reply contrib/JavaGraphqlClient/src/test/java/com/example/graphql/client/JavaGraphqlClient/JavaGraphqlClientApplicationTests.java:13]: I will add the testcase

**Comment by anshbansal** [comment]
Closing due to inactivity.

**Pull request closed without merging**